client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.tsx

```diff
@@ -159,20 +155,25 @@ function CheckoutSummaryFeaturesList( props: {
 		( product ) => isDomainProduct( product ) || isDomainTransfer( product )
 	);
 	const hasPlanInCart = responseCart.products.some( ( product ) => isPlan( product ) );
-	const translate = useTranslate();
-	const isJetpackNotAtomic = useSelector( ( state ) =>
-		siteId ? isJetpackSite( state, siteId ) && ! isAtomicSite( state, siteId ) : undefined
+	const refundableProducts = responseCart.products.filter(
```

Comment:
Displaying multiple rows for domain names with different TLDs looks like duplication. Could we group them, except for the case when the refund window varies.

Reply:
I see what you mean here. It does get a bit repetitive when there are multiple domains in the cart. 

If we want to do this, the main challenge as I see it has to do with product names and translations. The `product_name` property of domains varies depending on TLD, so simply grouping all domains might produce funny results. For example, if I have a `.ninja` and `.dev` in my cart, the order summary might say `4-day money back guarantee for .ninja Domain Registration`. This could get confounding for the customer ("isn't there a money back guarantee for my `.dev` domain?") 

One solution to this problem could be to always pass the string `Domain Registration` for domain products, instead of relying on `product.product_name`. I'm a little unsure about if this will reliably result in a translated product name, though... I will investigate!

In the meantime, I have pushed a fix that modifies the grouping logic.

Reply:
While grouping seems like a good idea, I think it's ok to keep them separate if it makes our logic too complex or fragile. We can have a look to data but I bet most users only purchase one domain at a time, so it probably doesn't worth the effort.